app/Donations.tsx

```diff
@@ -242,20 +244,45 @@ export default function Organizations() {
   useEffect(() => {
     applyQueryToFilters();
     return () => {};
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   useEffect(() => {
     history.pushState({}, "", applyFiltersToQuery());
     return () => {};
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [filters]);
 
   const filteredOrganizations: Organization[] = organizations.filter((org) =>
     isOrganizationFiltered(org)
   );
 
+  // Sort options by network
+  const sortByNetwork = (a: Option, b: Option) => {
+    if (a.type === "cryptocurrency" && b.type === "cryptocurrency") {
+      return (
+        NETWORKS.findIndex((network: Network) => network === a.name) -
+        NETWORKS.findIndex((network: Network) => network === b.name)
+      );
+    } else {
+      return 0;
+    }
+  };
+
+  // Option mutator
+  function sortCryptocurrenciesByNetworks(
+    organizations: Organization[]
+  ): Organization[] {
+    return [...organizations].map((organization: Organization) => {
```

Comment:
return [...organizations].map((organization) => organization.options.sort((a: any, b: any) => sortByNetwork(a, b)));

You can use this method like that to minify. But this is not a mandatory request. I said it for a suggestion.